includes/Plugin.php

```diff
@@ -141,16 +141,6 @@ function( $username, $user ) use ( $user_options ) {
 		( new Core\Util\Activation( $this->context, $options, $assets ) )->register();
 		( new Core\Util\Beta_Migration( $this->context ) )->register();
 		( new Core\Util\Migration_1_0_0( $this->context ) )->register();
-
-		if ( defined( 'WP_DEBUG' ) && true === WP_DEBUG ) {
-			add_filter(
-				'debug_bar_panels',
-				function( $panels ) {
-					$panels[] = new Core\Util\Debug_Bar();
-					return $panels;
-				}
-			);
-		}
```

Comment:
I realize this has been queued for removal but does this relate to the AC at all? I'm OK if we want to include it in this PR, just curious if there is overlap with the AC.

Reply:
As discussed earlier, we can remove this since the debug bar integration is not useful at all as it is.